ts/sagas/profile.ts

```diff
@@ -63,8 +63,47 @@ import {
   getLocalePrimaryWithFallback
 } from "../utils/locale";
 import { readablePrivacyReport } from "../utils/reporters";
+import { LollipopConfig } from "./../features/lollipop";
 
 // A saga to load the Profile.
+export function* loadProfileLollipop(
```

Comment:
it looks like this method shares code with the following `loadProfile`, maybe we can do a refactor to avoid duplication?

Reply:
as above